### PR TITLE
[REVIEW] Fix Python not being found by CMake

### PR DESCRIPTION
**Pull request by kkraus14**

### CODE REVIEW

File: conda/recipes/nvstrings/meta.yaml [hunk 1]
```diff
@@ -26,6 +26,7 @@ build:
 requirements:
   build:
     - cmake
+    - python
```
Comment (jakirkham) [inline conda/recipes/nvstrings/meta.yaml]: By default the version here won't be the same as in `host`. So we might need to do something like this...

```suggestion
    - python 3.6. # [py36]
    - python 3.7. # [py37]
```

Comment (kkraus14) [reply conda/recipes/nvstrings/meta.yaml]: Would specifying `python x.x` work for this?

Comment (jakirkham) [reply conda/recipes/nvstrings/meta.yaml]: Maybe? I'm not sure. That's sort of slowly getting phased out, but it's worth trying.

If not, this is a reasonable fallback. `x.x` should still work in `host` and `run` at least.

Comment (kkraus14) [reply conda/recipes/nvstrings/meta.yaml]: It does not work in build from the looks of it, will try your method

File: conda/recipes/nvstrings/meta.yaml [hunk 2]
```diff
@@ -26,17 +26,19 @@ build:
...
     - numba>=0.40
   run:
-    - python
+    - python x.x
     - setuptools
 
 test:
   requires:
-    - python
+    - python x.x
```
Comment (jakirkham) [inline conda/recipes/nvstrings/meta.yaml]: I think we can drop `x.x` here.

Comment (kkraus14) [reply conda/recipes/nvstrings/meta.yaml]: Can I just drop the Python entirely since that's a runtime requirement anyway?

Comment (jakirkham) [reply conda/recipes/nvstrings/meta.yaml]: Yep. Good point :)

**Pull request merged**